### DISCUSSION TITLE
@urql/core@3.0.4

### DISCUSSION THREAD

**Discussion by christian-schwan** [urql-graphql/urql#2728] [Q&A] [answered]
Can you please approve @urql/core@3.0.4?

**Comment by kitten** [comment, marked as answer]
No, we're currently awaiting more fixes for our next batch of releases. This is to minimise the amount of known bugs that we have to verify and fix per patch. It's common for us to wait for known bugs to all be resolved to issue a new version.

This has several reasons. It minimises the amount of feedback and subsequent investigation we may have to do. It minimises the amount of releases that users have to check to find changes that they're affected.

In short, while we're aware that people who are waiting for a specific fix (maybe because they've reported it) will be eager to see a fix that's been merged be released already, we always have to assume the vast majority of users is blissfully unaware of any issues, and in my eyes it's a win-win for them and us to delay releases sometimes.

Do note though that every single pull requests provides build artifacts. These are zipped up (due to a limitation in GH Actions) tarballs, that you can download, install, and commit (or otherwise use) if you want to avoid building any pre-release packages yourself: https://github.com/FormidableLabs/urql/actions/runs/3230726786#artifacts
